app/views/layouts/_footer.html.erb

```diff
@@ -30,6 +30,8 @@
         <span class="footer-link-pointer">> <%= link_to "#{t('footer.privacy')}", datenschutz_path, class: "center" %></span>
         <br>
         <span class="footer-link-pointer">> <%= link_to "#{t('footer.about')}", ueber_path, class: "center" %></span>
+        <br>
+        <span class="footer-link-pointer">> <%= link_to "Contact me", new_contact_url, class: "center" %></span>
```

Comment:
Schaue mal in Zeile 32. Da kannst du `_path`-Helper nehmen, wenn du Links innerhalb deiner Anwendung nutzt.

Reply:
Oh stimmt